vuepress/docs/next/tutorials/devops/installation/open-shift/openshift-install.md

```diff
@@ -4,6 +4,10 @@ sidebarDepth: 2
 
 # Installation on OpenShift
 
+:::tip
+It is highly recommended that users install via [OpenShift using the Operator Hub]. These instructions provide manual steps for achieving the same outcome
```

Comment:
Missing link? Duplicate of language in the next paragraph?

Reply:
Removed tip and added new wording in following paragraph